components/StatusItem/index.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { Text, View, StyleSheet } from 'react-native';
+import PropTypes from 'prop-types';
+import Icon from './../Icon';
+
+const StatusItemIcon = ({ danceStatus }) => {
+  if (danceStatus === 'Uploaded') {
+    return (
+      <Icon name="Checkmark" height="50" width="50" fill="grey" viewBox="0 0 30 30" />
+    );
+  } else if (danceStatus === 'Loading') {
+    return (
+      <Icon name="Loop" height="50" width="50" fill="black" viewBox="0 0 30 30" />
+    );
+  } else if (danceStatus === 'Required internet connection') {
+    return (
+      <Icon name="Exclamation" height="50" width="50" fill="red" viewBox="0 0 30 30" />
+    );
+  }
+  return (null);
+};
+
+StatusItemIcon.propTypes = {
+  danceStatus: PropTypes.string.isRequired,
+};
+
+const Information = ({ danceNum, danceName }) => (
+  <View style={style.information}>
+    <Text style={style.danceNum}>#{danceNum}</Text>
+    <Text style={style.danceName}>{danceName}</Text>
+  </View>
+);
+
+
+Information.propTypes = {
+  danceNum: PropTypes.number.isRequired,
+  danceName: PropTypes.string.isRequired,
+};
+
+const StatusItem = ({ danceNum, danceName, danceStatus }) => (
```

Comment:
I don't think you need to have a separate Information and StatusItemIcon component since they are only used in this file. It'll be cleaner to just combine everything instead of having to pass the same props around.

Reply:
yo tru